kahuna/public/js/components/gr-add-label/gr-add-label.css

```diff
@@ -1,3 +1,12 @@
+gr-add-label{
+    display: flex;
+    height: 25px;
+}
+
+gr-add-label form input {
+    width: 150px;
```

Comment:
Styling html elements directly should be avoided if it can, perhaps class/name up the input and specifically target it.